src/utils/authentication/firebase.js

```diff
@@ -0,0 +1,16 @@
+import firebase from 'firebase';
+
+const API_SECRET_KEY = process.env.REACT_APP_FB_API_KEY;
```

Comment:
Even though the custom login and registration works, remember that react is a client side application, which means that for it to connect to your firebase instance it needs to have access to that key. What I'm trying to say is that don't rely only on having your secrets in the `.env` file, but implement security also on the server side (firebase in this case), and only allow certain domains so that you restrict its usage. Haven't tested if this was your case, but just a FYI for future implementations.